### PR TITLE
[5] admin module deprecation

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #41605

### Summary of Changes
do the check only if not null

### Testing Instructions
see #41605
php 8.1

### Actual result BEFORE applying this Pull Request

deprecation 

### Expected result AFTER applying this Pull Request

no more deprecation

### CODE REVIEW

File: libraries/src/Table/Module.php [hunk 1]
```diff
@@ -150,11 +150,14 @@ public function check()
             $this->publish_down = null;
         }
 
-        // Prevent to save too large content > 65535
-        if ((\strlen($this->content) > 65535) || (\strlen($this->params) > 65535)) {
-            $this->setError(Text::_('COM_MODULES_FIELD_CONTENT_TOO_LARGE'));
+        if (!\is_null($this->content) && !\is_null($this->params)) {
```
Comment (Quy) [inline libraries/src/Table/Module.php]: `content` is null and `params` is not so `params` check will be skipped. 

It should be the following:
`if ((!\is_null($this->content) && \strlen($this->content) > 65535) || (!\is_null($this->params) && \strlen($this->params) > 65535)) {`

Comment (alikon) [reply libraries/src/Table/Module.php]: right

File: libraries/src/Table/Module.php [hunk 2]
```diff
@@ -150,11 +150,13 @@ public function check()
...
 
-        // Prevent to save too large content > 65535
-        if ((\strlen($this->content) > 65535) || (\strlen($this->params) > 65535)) {
+        if ((!\is_null($this->content) && \strlen($this->content) > 65535) || (!\is_null($this->params) && \strlen($this->params) > 65535)) {
+
+            // Prevent to save too large content > 65535
             $this->setError(Text::_('COM_MODULES_FIELD_CONTENT_TOO_LARGE'));
 
             return false;
+            }
```
Comment (Quy) [inline libraries/src/Table/Module.php]:
```suggestion
```

File: libraries/src/Table/Module.php [hunk 3]
```diff
@@ -150,11 +150,13 @@ public function check()
             $this->publish_down = null;
         }
 
-        // Prevent to save too large content > 65535
-        if ((\strlen($this->content) > 65535) || (\strlen($this->params) > 65535)) {
+        if ((!\is_null($this->content) && \strlen($this->content) > 65535) || (!\is_null($this->params) && \strlen($this->params) > 65535)) {
+
+            // Prevent to save too large content > 65535
```
Comment (Quy) [inline libraries/src/Table/Module.php]: Please move comment back as before.

Comment (alikon) [reply libraries/src/Table/Module.php]: done

**Comment by HLeithner** [comment]
I would say an !empty would fit better here at least use !== null

**Comment by HLeithner** [comment]
thanks

**Pull request merged**